docs/source/interfaces/muon_transform_tab.rst

```diff
@@ -16,27 +16,24 @@ In this mode the interface displays two tables, FFT and Advanced options. The FF
 all of the information required to produce a forward FFT. The Advanced Options table contains the information
 for adding padding and apodization functions to the data.
 
-Selecting the workspace to be ``PhaseQuad`` will cause the :ref:`PhaseQuad <algm-PhaseQuad>` algorithm to be called first. If no phase table is available then
-the interface will generate one and output the result as ``PhaseTable``. It is possible to select the direction of the axis when generating the table and it is
-possible to use a pre-existing ``PhaseTable``.
+If the workspace contains the word ``PhaseQuad`` it will run the :ref:`PhaseQuad <algm-PhaseQuad>` algorithm on the selected run when ``Calculate`` is pressed. If no phase table is available then
+the one will be generated and the result will be stored as ``PhaseTable``.
 
-The Calculate button will execute the :ref:`Padding and Apodization <algm-PaddingAndApodization>`  and then :ref:`FFT <algm-FFT>` algortihms. The output workspace will be added to
+The ``Calculate`` button will execute the :ref:`Padding and Apodization <algm-PaddingAndApodization>`  and then :ref:`FFT <algm-FFT>` algorithms. The output workspace will be added to
```

Comment:
Here you have `unformatted text` for "Calculate" but later on you have paragraph formatting for the same text in the same context.

One style should be used for all references to UI elements.